test/e2e/memcached_test.go

```diff
@@ -479,23 +479,36 @@ func memcachedScaleTest(t *testing.T, f *framework.Framework, ctx *framework.Tes
 		t.Fatalf("Failed to unmarshal memcached CR: (%v)", err)
 	}
 	obj.SetNamespace(namespace)
-	err = f.Client.Get(context.TODO(), types.NamespacedName{Name: obj.GetName(), Namespace: obj.GetNamespace()}, &obj)
-	if err != nil {
-		return fmt.Errorf("failed to get memcached object: %s", err)
-	}
-	// update memcached CR size to 4
-	spec, ok := obj.Object["spec"].(map[string]interface{})
-	if !ok {
-		return errors.New("memcached object missing spec field")
-	}
-	spec["size"] = 4
-	err = f.Client.Update(context.TODO(), &obj)
+
+	err = wait.Poll(retryInterval, timeout, func() (done bool, err error) {
+		err = f.Client.Get(context.TODO(), types.NamespacedName{Name: obj.GetName(), Namespace: obj.GetNamespace()}, &obj)
+		if err != nil {
+			return false, fmt.Errorf("failed to get memcached object: %s", err)
+		}
+		// update memcached CR size to 4
+		spec, ok := obj.Object["spec"].(map[string]interface{})
+		if !ok {
+			return false, errors.New("memcached object missing spec field")
+		}
+		spec["size"] = 4
+		err = f.Client.Update(context.TODO(), &obj)
+		if err != nil {
+			if apierrors.IsConflict(err) {
+				return false, nil
```

Comment:
Log that we're retrying on conflict like we do below for test-framework e2e test:
```suggestion
				t.Logf("Update conflict for example-memcached, retrying.")
				return false, nil
```